src/pages/Dragons.js

```diff
@@ -1,25 +1,34 @@
-import React, { useEffect } from 'react';
+import React from 'react';
 import { useDispatch, useSelector } from 'react-redux';
-import { cancelReservation, dragonAction, reverseDragonAction } from '../redux/dragons/dragon';
+import toast from 'react-hot-toast';
+import { cancelReservation, reverseDragonAction } from '../redux/dragons/dragon';
 import DragonContainer from '../components/DragonsContainer';
-import fetchDragons from '../api/dragonAPI';
 
 export default function Dragonsdisplay() {
   const dragons = useSelector((state) => state.dragons);
   const dispatch = useDispatch();
-  useEffect(() => {
-    async function data() {
-      const dragons = await fetchDragons();
-      dispatch(dragonAction(dragons));
-    }
-    data();
-  }, []);
-  const handleReserve = (id) => {
+  const toastDisplay = (message) => {
+    toast.success(message, {
+      style: {
+        border: '1px solid blue',
+        padding: '16px',
+        color: 'blue',
+      },
+      iconTheme: {
+        primary: 'blue',
+        secondary: '#FFFAEE',
+      },
```

Comment:
please it could be nice if you add your styling externally   inline CSS makes your code a bit messy

Reply:
Hey @Bria222 , 
As you can see the style Is actually the second argument of the method `toast.success`, there's could be no way to implement it in a  `scss` or `css` file.